tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs

```diff
@@ -61,14 +62,30 @@ public async Task GetAccessTokenForApp_ReturnsAccessTokenAsync()
 
             // Act
             string token =
-                await _tokenAcquisition.GetAccessTokenForAppAsync(TestConstants.s_scopesForApp).ConfigureAwait(false);
+                await _tokenAcquisition.GetAccessTokenForAppAsync(TestConstants.s_scopeForApp).ConfigureAwait(false);
 
             // Assert
             Assert.NotNull(token);
 
             AssertAppTokenInMemoryCache(TestConstants.ConfidentialClientId, 1);
         }
 
+        [Fact]
```

Comment:
you can do different arguments and replace the tenant value w/organziations, common, consumers.
```csharp
[Theory]
[InlineData(Constants.Common)]
[InlineData(Constants.Organizations)]
[InlineData(Constants.Consumers)]
public async Task GetAccessTokenForApp_WithMetaTenant(string metaTenant)
{
....
await _tokenAcquisition.GetAccessTokenForAppAsync(TestConstants.s_scopeForApp, metaTenant).ConfigureAwait(false);
....
}

Reply:
We do already with the test that uses user scopes.

Reply:
Addressed